srvrepo/srvrepo.go

```diff
@@ -118,6 +118,19 @@ func NewServerRepository() *ServerRepository {
 	}
 }
 
+// Check if an given ServerID already exists in the repository
+func (r *ServerRepository) Has(id ServerID) (bool, error) {
+	alreadyExists := false
+	var err error
```

Comment:
Does it make sense to return an error if we're always going to return the same thing?